templates/bundles/DemosPlanCoreBundle/DemosPlanUser/password_recover.html.twig

```diff
@@ -19,12 +19,14 @@
                     control: { name: 'email' },
                     id: 'email',
                     type: 'email',
+                    dataCy: 'passwordRecoverForm:email',
```

Comment:
I'd suggest to pass the attribute via `attributes` within the `control` field. See templates/bundles/DemosPlanCoreBundle/DemosPlanFaq/faq_admin_category_edit.html.twig:21 for reference.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/1742/commits/60c5c909b95dfc5f604b18f71bbec1c4c4a6755b
Thanks!